zanata-war/src/main/webapp/WEB-INF/pages.xml

```diff
@@ -262,8 +246,9 @@
     <action execute="#{breadcrumbs.addLocation('', msgs['jsf.Groups'])}" />
   </page>
 
-  <page view-id="/version-group/create_version_group.xhtml">
-    <restrict>#{s:hasPermission(versionGroupHome.instance, 'insert')}</restrict>
+  <!--TODO [CDI] we need to find a way to secure this page's access in CDI (the actual perist method is already secured) -->
+  <page view-id="/version-group/create_version_group.xhtml"
+    action="#{identity.checkPermission(versionGroupHome.instance, 'insert')}">
```

Comment:
Could we split up VersionGroupHome, put all the mutating operations into their own class, and make sure only the group maintainer pages use the mutating bean?